src/go/configmanager/main/server.go

```diff
@@ -74,6 +77,15 @@ func main() {
 		grpcServer.Stop()
 	}()
 
+	if opts.ServiceAccountKey != "" {
+		// Setup local service-account-generated token server
+		r := tokengenerator.MakeLatsHandler(opts.ServiceAccountKey)
+		go func() {
+			_ = http.ListenAndServe(fmt.Sprintf(":%v", opts.LocalAccessTokenServerPort), r)
```

Comment:
You're ignoring the error here. We have to handle it.

Are you sure this needs to be in a goroutine? I thought `ListenAndServe` would handle the blocking call internally.

Reply:
ListenAndServe is blocking 
https://blog.zikes.me/post/serving-multiple-sites-from-one-go-process/

Reply:
Then I recommend checking the error and logging it, it will help debugging when a shutdown occurs.

Recommend using `glog.Errorf(...)` here.